### PR TITLE
refactor(ui): Polish appearance of Alerting UI

### PR DESCRIPTION
**Pull request by alexpaxton**
Screen Shot 2019-08-08 at 3 35 15 PM
alert-check-builder

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/alerting/components/ChecksColumn.tsx [hunk 1]
```diff
@@ -25,19 +26,18 @@ const ChecksColumn: FunctionComponent<Props> = ({
...
 const mstp = (state: AppState) => {
-  const {
-    checks: {list: checks},
-  } = state
+  // const {
+  //   checks: {list: checks},
+  // } = state
 
-  return {checks}
+  return {checks: [check1, check2]}
```
Comment (ebb-tide) [inline ui/src/alerting/components/ChecksColumn.tsx]: ah, maybe we don't need to check this in?

File: ui/src/alerting/components/EndpointsColumn.tsx [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
+      testID="create-endpoint"
+      onCreate={() => {}}
+    >
+      <EmptyState
+        size={ComponentSize.ExtraSmall}
+        className="alert-column--empty"
+      >
+        <EmptyState.Text
+          text="Looks like you don’t have any Checks , why not create one?"
+          highlightWords={['Checks']}
```
Comment (ebb-tide) [inline ui/src/alerting/components/EndpointsColumn.tsx]: Could this say endpoints? ;)

File: ui/src/alerting/components/ChecksColumn.tsx (hunk 1 above)
Comment (alexpaxton) [reply ui/src/alerting/components/ChecksColumn.tsx]: Wooooooops, fixing now!

File: ui/src/alerting/components/EndpointsColumn.tsx (hunk 2 above)
Comment (alexpaxton) [reply ui/src/alerting/components/EndpointsColumn.tsx]: YES

File: ui/src/alerting/components/AlertingIndex.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
+.alerting-index {
+  > .page-contents > .container-fluid,
+  > .page-contents > .container-fluid > .row,
+  > .page-contents > .container-fluid > .row > .col-xs-12,
+  > .page-contents > .container-fluid > .row > .col-xs-12 > .cf-grid--container,
+  > .page-contents > .container-fluid > .row > .col-xs-12 > .cf-grid--container > .cf-grid--row,
+  > .page-contents > .container-fluid > .row > .col-xs-12 > .cf-grid--container > .cf-grid--row > .cf-grid--column {
```
Comment (alexpaxton) [inline ui/src/alerting/components/AlertingIndex.scss]: 😬

**Pull request merged**